kimyeongseok/db/migrations/20221124055617_create_users_table.sql

```diff
@@ -0,0 +1,4 @@
+-- migrate:up
+ALTER TABLE users MODIFY updated_at TIMESTAMP
```

Comment:
```suggestion
ALTER TABLE users MODIFY updated_at TIMESTAMP NULL on update DEFAULT CURRENT_TIMESTAMP
```

`updated_at` 컬럼은 수정이 되는 그 시점의 시간이 데이터베이스에 저장되도록 수정해주세요!